platforms/unix/src/adapter.rs

```diff
@@ -157,8 +154,14 @@ impl Adapter {
             },
             "accesskit_event_task",
         );
-        let tree = Tree::new(initial_state(), is_window_focused);
-        let app_context = AppContext::new(app_name, toolkit_name, toolkit_version);
+        let initial_state = initial_state();
+        let initial_tree = initial_state.tree.as_ref().unwrap();
+        let app_context = AppContext::new(
+            initial_tree.app_name.clone().unwrap_or_default(),
```

Comment:
I don't really know why would someone do this, but we allow changing these values at any time. To be accurate, we should check on each tree update if the values are provided, and update them accordingly.

Reply:
I have considered removing the fields from the app_context, since they are now on the tree anyway. What do you think about that? We can just always get them from the current Tree instance and then we wouldn't have to care about updating the context. just a thought though.

Reply:
This is actually colliding with #292.
What if two adapters push different values? @mwcampbell

Reply:
I think the best we can do in this case is what #292 already does; the first set of values wins.

Reply:
Sorry, I had overlooked the context of the comment. For the Unix adapter, I think we want to keep the fields on the app context, and copy them there from the first tree update when the app context is created.

Reply:
Copying the values at initialization time is exactly what happens now, right? I'm not sure whether I should still change anything here.

Reply:
No you're good.